litestar/middleware/compression.py

```diff
@@ -173,8 +173,8 @@ def create_compression_send_wrapper(
         bytes_buffer = BytesIO()
         facade = CompressionFacade(buffer=bytes_buffer, compression_encoding=compression_encoding, config=self.config)
 
-        initial_message = Ref[Optional["HTTPResponseStartEvent"]](None)
```

Comment:
Again, I don't know why this was introduced here in the first place other than that it was. I also don't see a reason why simply using `nonlocal` wouldn't work here so I went with that.